### DISCUSSION TITLE
Branding

### DISCUSSION THREAD

**Discussion by 3ill** [Mchango/mchango#12] [General]
We need to establish brand identity, this is supposed to be our first step, as you can tell the logo is just a placeholder and our readme logo is a place holder also. Please let's get this done as soon as possible. Thank you 🙏🏾

**Comment by nvsblmike** [comment]
very true. In terms of the smart contract, what needs to be tacked first?
So as not to feel overwhelmed. Can you list them in terms of scale of preference?

**Reply by 3ill** [reply]
Check the projects section, you'll see the Todos for the smart contract

**Reply by 3ill** [reply]
If you are making updates to the code, create a branch, don't push to the Main repo

**Reply by nvsblmike** [reply]
No problem.
